src/components/App/Select/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { fetchEpisodes } from 'redux/actions/episodesActions';
+import PropTypes from 'prop-types';
+
+import styles from './Select.module.scss';
+
+class Select extends React.Component {
+  render() {
+    return (
+      <select onChange={e => this.props.onChange(e.target.value)} value={this.props.value} className={styles.customSelect}>
+        {this.props.options.map(option => (
+          <option value={option.code} key={option.code}>
+            {option.name}
+          </option>
+        ))}
+      </select>
+    );
+  }
+}
+
+const mapDispatchToProps = dispatch => (
```

Comment:
Why do you need this action here?

Reply:
Well, you're right again. :) Actually it seems that Select doesn't have to be a container at all, I'm transforming it back into a regular component.